install/sessions.csv

```diff
@@ -656,7 +656,8 @@ WGTN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2
 WGTN,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2020-02-14T01:16:38Z,9999-01-01T00:00:00Z
 WGTT,GeoNet,GNS,LEICA CRS1000,GPS,30s,4,geonet,leica_lb2 x6,1999-12-14T00:00:00Z,2006-08-24T01:20:00Z
 WGTT,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2006-08-24T01:20:30Z,2019-03-11T01:44:11Z
-WGTT,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2019-03-11T01:44:12Z,9999-01-01T00:00:00Z
+WGTT,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2019-03-11T01:44:12Z,2024-07-15T01:21:19Z
+WGTT,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2024-07-15T01:21:20Z,9999-01-01T00:00:00Z
```

Comment:
We've updated to full constellation tracking `GPS+GLO+GAL+BDS+QZSS` for the GNSS at tide gauge project for LINZ.

Reply:
Changed. See latest commit.